src/containers/share-modal/components/DesktopShareModal.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { MouseEventHandler, ReactNode } from 'react'
+
+import {
+  Button,
+  ButtonProps,
+  ButtonType,
+  IconLink,
+  IconShare,
+  IconTikTok,
+  IconTwitterBird,
+  Modal
+} from '@audius/stems'
+
+import styles from './DesktopShareModal.module.css'
+
+type ShareActionListItemProps = ButtonProps
+
+const ShareActionListItem = (props: ShareActionListItemProps) => {
+  return (
+    <li className={styles.actionListItem}>
+      <Button
+        className={styles.actionButton}
+        type={ButtonType.COMMON_ALT}
+        {...props}
+      />
+    </li>
+  )
+}
+
+type DesktopShareModalProps = {
+  onShareToTwitter: () => void
+  onShareToTikTok: () => void
+  onCopyLink: () => void
+  isOpen: boolean
+  onClose: () => void
+  isOwner: boolean
+}
+
+export const DesktopShareModal = ({
+  onShareToTwitter,
+  onShareToTikTok,
+  onCopyLink,
+  isOpen,
+  onClose,
+  isOwner
+}: DesktopShareModalProps) => {
+  return (
+    <Modal
+      allowScroll={false}
+      bodyClassName={styles.modalBody}
+      headerContainerClassName={styles.headerContainer}
+      isOpen={isOpen}
+      onClose={onClose}
+      showTitleHeader
+      showDismissButton
+      title={
+        <h2 className={styles.titleContainer}>
+          <IconShare />
+          Share Track
+        </h2>
+      }
+    >
+      <div className={styles.modalContent}>
+        <p className={styles.description}>
+          Spread the word! Share with your friends and fans!
```

Comment:
Let's put all copy into a `messages` object. The hope is that it will help us with i18n in the future